README.md

```diff
@@ -1,56 +1,31 @@
-# licence-finder
+# Licence Finder
 
-Licence finder provides the pages that follow https://www.gov.uk/licence-finder (which is served by `frontend`), specifically starting at https://www.gov.uk/licence-finder/sectors.
+Licence Finder is a public facing app that provides the pages that follow https://www.gov.uk/licence-finder (which is served by [frontend][]), specifically starting at https://www.gov.uk/licence-finder/sectors.
 
-## Local setup
+[frontend]: https://github.com/alphagov/frontend
 
-To get set up with Licence Finder, first make sure you're up to date with puppet.
+## Technical documentation
 
-Load the data into MongoDB
+This is a Ruby on Rails app, and should follow [our Rails app conventions](https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html).
 
-    bundle exec rake data_import:all
+You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-docker) to run the application and its tests with all the necessary dependencies. Follow [the usage instructions](https://github.com/alphagov/govuk-docker#usage) to get started.
 
-Load the data into Elasticsearch
+**Use GOV.UK Docker to run any commands that follow.**
 
-    bundle exec rake search:index
+## Before running the app
 
-Migrating your Licence data to use gds_id instead of correlation_id (This happens automatically on importing licences)
+The first time you run this application in development you will want to populate the search index. This can be done with the following command:
 
-    bundle exec rake licence_migrate
-
-### Running the application
-
-Using bowler on the VM from cd /var/govuk/development/:
-
-```
-bowl licencefinder
 ```
-
-If you are using the GDS development virtual machine then the application will be available on the host at http://licencefinder.dev.gov.uk/licence-finder
-
-#### Without using the VM
-
-Licence Finder can be run against the live Content Store and Search API, however it needs local instances of Elasticsearch and MongoDB.
-
-You can use docker to create these:
-
-```
-docker run -d -p 9200:9200 -e "discovery.type=single-node" --name licence-finder-elasticsearch docker.elastic.co/elasticsearch/elasticsearch:6.8.0
-docker run -d -p 27017-27019:27017-27019 --name licence-finder-mongodb mongo:latest
+bundle exec rake search:index
 ```
 
-Import the data and start the application:
+### Running the test suite
 
 ```
-export ELASTICSEARCH_URI='http://localhost:9200/'
-export MONGODB_URI='mongodb://localhost/'
-bundle exec rake data_import:all
```

Comment:
I worry that we may be losing some useful documentation here, as we're removing this content from the README but not removing the `data_import:all` task itself, which would otherwise be left undocumented.

Perhaps we could move this to a separate `docs/live-data.md`, and an introduction added to it which basically caveats it with "this task is unreliable and may hang"?

Reply:
It seems a bit of a can of worms to try do anything with this documentation since the expectation is that the task won't work for the user. 

I'm happy for it to be documented just by what's provided with the rake task. I'm not expecting anyone to really run it though, it'd seem far easier to export a database dump.